### PR TITLE
improve scaling example

### PR DESCRIPTION
**Pull request by james-powis**
The example for `Multiple APIs with reusable namespaces` is incomplete and using variables that clash with previous examples on the same page.

This PR aims to provide a complete example by providing an example of a blueprint that can be used by the established documentation.

### CODE REVIEW

**Comment by james-powis** [comment]
This change was purely documentation, there were no code changes. Looks like this failure is related to something else.

**Comment by j5awry** [comment]
the benchmark run is failing generally. That's something on our side and should be non-blocking.

File: doc/scaling.rst [hunk 1]
```diff
@@ -214,6 +214,19 @@ Given the previous layout, we can migrate it to the following directory structur
...
+Each `apis/namespaceX` module will have the following pattern:
+
+.. code-block:: python
+
+    from flask_restplus import Namespace
+
+    api = Namespace('mynamespace', 'Namespace Description' )
+
+    @api.route("/")
+    class Myclass(Resource):
```
Comment (j5awry) [inline doc/scaling.rst]: please include the Resource import

`from flask-restx import Resource`

Comment (james-powis) [reply doc/scaling.rst]: good catch, this has been done

**Pull request merged**